spec/features/user_cannot_access_admin_functionalities_spec.rb

```diff
@@ -17,6 +17,7 @@
     expect(page).to_not have_content("Delete Venue")
     expect(page).to_not have_content("Delete Review")
 
-    expect { visit admin_users_path }.to raise_error
+    expect { visit admin_users_path
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Block body expression is on the same line as the block start.